### DISCUSSION TITLE
How to try v4

### DISCUSSION THREAD

**Discussion by xo-o** [juliangarnier/anime#896] [Q&A] [answered]
Seems like I'm sponsoring the project now but I am not able to see any information related to v4 in the repo. does anyone know how I can try v4?

**Comment by juliangarnier** [comment, marked as answer]
Hello and thank you for your one time sponsor!

I’m sorry if this wasn’t clear enough from the sponsor page, but GitHub only allows automatic access to private repositories to monthly sponsors. And I don’t think I can prevent people from sponsoring me for one time payment.

Only the listed monthly tiers on the sponsor page mention the V4 beta access.

Although, it’s totally possible to sponsor the v4 beta repository for one month, access and download the full code and documentation and cancel the subscription before the next month, but it has to be a monthly sponsorship of 10$ or higher.

Hope this helps and sorry again if it wasn’t clear enough on the sponsor page!

**Reply by xo-o** [reply]
Got it, I have access now. Updated sponsorship tier.

**Reply by juliangarnier** [reply]
Thank you, and hope you enjoy v4! 
Come say hi in the Discord if you feel like it!

**Reply by xo-o** [reply]
I could not find the discord link, seems like it is in a secret place. Could you share it ?

**Reply by juliangarnier** [reply]
You must have received an email after sponsoring, with a link to the Discord server.

**Reply by juliangarnier** [reply]
I also just updated the readme file of the private beta repo with the link to the Discord server.

**Comment by dinther** [comment]
It's a grift. This project has been in beta forever. 
I can't help but think that in this case eternal beta is quite profitable.

So... I gave up and wrote my own from scratch including UI

![image](https://github.com/user-attachments/assets/753312e4-2920-49df-9830-865e43dddaf3)

**Reply by different55** [reply]
Coming into someone else's repo, calling them a grifter, and trying to push your own repo. Classy move.

**Reply by juliangarnier** [reply]
> It's a grift. This project has been in beta forever. I can't help but think that in this case eternal beta is quite profitable.
> 
> So... I gave up and wrote my own from scratch including UI

I'm so sorry you had to wait this long. It was indeed very hard for me to get out of the very profitable early access.
But at least now I can read all the nice comments and encouragements people have written while waiting for the public release.

**Comment by juliangarnier** [comment]
Closing this now that v4 is out!!
